packages/cxl-ui/src/components/cxl-light-card.js

```diff
@@ -1,17 +1,40 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import { html, nothing } from 'lit';
+import { html, LitElement, nothing } from 'lit';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
-import { customElement, property } from 'lit/decorators.js';
-import '@conversionxl/cxl-lumo-styles';
+import { customElement, property, state } from 'lit/decorators.js';
+
+import '@vaadin/progress-bar';
+
 import cxlLightCardStyles from '../styles/cxl-light-card-css.js';
-import { CXLBaseCardElement } from './cxl-base-card.js';
 
 @customElement('cxl-light-card')
-export class CXLLightCardElement extends CXLBaseCardElement {
+export class CXLLightCardElement extends LitElement {
```

Comment:
@HenerHoop  you missed this, and now it looks like you copied a lot of what base-card already handles. This is becoming way too complicated for a component that doesn't even have logic. We got lost here.

Reply:
I'm sorry, @HenerHoop, looks like I forgot to submit the old review. I take responsibility for that. But it did do exactly the opposite of what was agreed in a call with Hesh, so that's still a problem. We need this to go forward, and it looks to be working well now, so let's push it over the line and add a backlog task to fix it later.